utils/lists.py

```diff
@@ -10,6 +10,22 @@
 _all_questions = [
     QuestionAnswer("What is the capital city of Spain?", "Madrid"),
     QuestionAnswer("What is arachnophobia the fear of?", "Spiders"),
+    QuestionAnswer("Stockholm is the capital and largest city of what country?", "Sweden"),
+    QuestionAnswer("When did the website Facebook launch?", "2004"),
+    QuestionAnswer("T/F: The scientific name for the Southern Lights is Aurora Australis?", "T"),
+    QuestionAnswer("On average, Americans consume 100 pounds of what per second?", "Chocolate"),
+    QuestionAnswer("How many bones are there in the human body?", "206"),
+    QuestionAnswer("What does Na stand for on the periodic table?", "Sodium"),
+    QuestionAnswer("Which fictional city is the home of Batman?", "Gotham"),
+    QuestionAnswer("Which planet is closest to Earth?", "Venus"),
+    QuestionAnswer("In the US version of The Office, what is the name of the city they live in?", "Scranton"),
+    QuestionAnswer("Which U.S. State is the largest?", "Alaska"),
```

Comment:
Largest in what category? Land area, population, something else? (The answer makes that obvious, but put yourself in the shoes of a person answering the question.) "Largest" is commonly used for both land area and population.